Makefile

```diff
@@ -5,7 +5,7 @@
 update-deps:
 	pip install --upgrade pip-tools pip setuptools
 	pip-compile --upgrade --build-isolation --allow-unsafe --generate-hashes --output-file requirements/main.txt requirements/main.in
-	pip-compile --upgrade --build-isolation --allow-unsafe --generate-hashes --output-file requirements/dev.txt requirements/dev.in
+	pip-compile --upgrade --build-isolation --allow-unsafe --generate-hashes --output-file requirements/dev.txt requirements/dev.in requirements/main.in
```

Comment:
Something I just recently learned to solve this issue is that you can add a `-c main.txt` line to the top of the `dev.in` requirements file to ensure that `dev.txt` is computed consistently with `main.txt`. I've meant to push that into the package template but just realized it during the `nb_lsst_io` work.

Reply:
Oh, that seems better. Thanks, I'll switch to that.